### DISCUSSION TITLE
How to add path to image from database?

### DISCUSSION THREAD

**Discussion by bartclaeys** [filamentphp/filament#4896] [Help]
How can I display an image from the database that does not include a path?

Eg. my image: `https://www.cloudflarecdn.com/3187d712-6158-11ed-9b6a-0242ac120002`

What's in the database (uuid): `3187d712-6158-11ed-9b6a-0242ac120002`

This unfortunately doesn't work: `ImageColumn::make('uuid')->getImagePath('http://www.cloudflarecdn.com/')`

**Comment by danharrin** [comment]
`getImagePath()` should only be used internally

ensure the storage disk is set up correctly to point to the cloudflare URL

**Reply by bartclaeys** [reply]
Thanks, I've looked into `disk()` but it doesn't allow me to append something to the URL. How would I go about that?
